signac/contrib/job.py

```diff
@@ -263,6 +276,34 @@ def update_statepoint(self, update, overwrite=False):
         statepoint.update(update)
         self.reset_statepoint(statepoint)
 
+    def _read_manifest(self):
+        """Read and parse the manifest file, if it exists.
+
+        Returns
+        -------
+        manifest : dict
+            State point data.
+
+        Raises
+        ------
+        JobsCorruptedError
+            If an error occurs while reading/parsing the state point manifest.
+        OSError
+            If an error occurs while reading/parsing the state point manifest.
+
+        """
+        fn_manifest = os.path.join(self._wd, self.FN_MANIFEST)
+        try:
+            with open(fn_manifest, "rb") as file:
+                manifest = json.loads(file.read().decode())
+        except OSError as error:
+            if error.errno != errno.ENOENT:
+                raise error
```

Comment:
Does this need to be tested?

Reply:
This `OSError` is not something I know how to test. This is probably raised by things like hardware issues, insufficient filesystem privileges, or dropped network connections.